linux/ibus-keyman/build.sh

```diff
@@ -33,6 +33,8 @@ else
 fi
 MESON_PATH="../build/$(uname -m)/$MESON_TARGET"
```

Comment:
Do you want to define MESON_PATH relative to THIS_SCRIPT_PATH? Then you could leave the other lines intact

Reply:
For performance reasons, it is good to make `builder_describe_outputs` absolute (i.e. to root of repo). I notice that it's missing from build-utils.md at present....